core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -201,9 +201,9 @@ class TopicCommandTest {
       () => topicService.deleteTopic(new TopicCommandOptions(Array("--topic", topicName))))
     assertTrue(exception.getCause.isInstanceOf[ThrottlingQuotaExceededException])
 
-    verify(adminClient, times(1)).deleteTopics(
-      argThat((_.contains(topicName)): ArgumentMatcher[Collection[String]]),
-      argThat((_.shouldRetryOnQuotaViolation() == false): ArgumentMatcher[DeleteTopicsOptions])
+    verify(adminClient).deleteTopics(
+      argThat((topics: java.util.Collection[String]) => topics.asScala.toBuffer.equals(Seq(topicName))),
```

Comment:
Do we know why ccs had "contains" rather than equals?

Reply:
The upstream change is the right one, a slightly different change had been applied to ccs which made the test assertion unnecessarily weaker.